sites/page-details/includes/footer.html

```diff
@@ -1,27 +1,25 @@
 <section class="pagedetails{% if include.addContainer %} container{% endif %}">
 	<h2 class="wb-inv">{{ i18nText-pageDetails }}</h2>
 
-	{%- if page.noReportProblem -%}
+	{% assign col-modified = "col-xs-12" %}
+	{%- if page.noFeedback != true -%}
+		{% assign col-feedback = "col-sm-8 col-md-9 col-lg-9" %}
```

Comment:
It seem to contain a behaviour change which would had an impact on the component, 

TODO:
[ ] need to deeply review the change made at this file.